piwik.php

```diff
@@ -82,26 +84,28 @@
 @ini_set('magic_quotes_runtime', 0);
 
 // 2) PIWIK.PHP PROXY: GET parameters found, this is a tracking request, we redirect it to Piwik
-$url = sprintf("%spiwik.php?cip=%s&token_auth=%s&", $PIWIK_URL, getVisitIp(), $TOKEN_AUTH);
-
-foreach ($_GET as $key => $value) {
-    $url .= urlencode($key ). '=' . urlencode($value) . '&';
+if (!isset($path)) {
```

Comment:
We may need to init `$path` initially. I'm thinking of older PHP versions where it is possible to say like `?path=...` and then it is available as a variable. It's not a huge risk but it would allow users to potentially call any file which may be for example IP restricted. But because the tracker proxy may be running in the same network etc the request would be allowed. As said, it is not really a big issue, but better be safe. May need to have a another file that is not directly executable which both `configs.php` and `piwik.php` can include or so.

Reply:
An init.php file sounds good, just in case someone creates an executable config.php or something.

Reply:
Setting `$path` in piwik.php (either directly or indirectly) doesn't work since it would overwrite the value set in, eg, configs.php. Solved this by renaming piwik.php => proxy.php & including that w/ `$path` set. So this way, it's never uninitialized.